lua/luasnip/loaders/from_snipmate.lua

```diff
@@ -0,0 +1,214 @@
+---@diagnostic disable: redefined-local
+local ls = require("luasnip")
+local caches = require("luasnip.loaders._caches")
+local util = require("luasnip.util.util")
+local Path = require("luasnip.util.path")
+local session = require("luasnip.session")
+
+local function parse_snipmate(buffer, filename)
+	local snippet = {}
+	local extends = {}
+
+	---@type string[]
+	local lines = vim.split(buffer, "\n")
+	if lines[#lines] == "" then
+		table.remove(lines)
+	end
```

Comment:
Maybe I overlooked it, but are commented lines also filtered? If not, this would be a good place to remove them I think.
Also, but rather minor, I think it would be faster to just append the non-filtered lines to a new list, `remove` "moves down" all following lines.

Reply:
Comments are handled in line 60. I don't need this because I'm also handling blank lines there.

Reply:
Ahh okay, didn't get to that part yet :sweat_smile: